travis/test_server.py

```diff
@@ -352,15 +353,15 @@ def main(argv=None):
         print("WARNING!\nNothing to test- exiting early.")
         return 0
     else:
-        print("Modules to test: %s" % tested_addons)
-    # setup the base module without running the tests
+        print("Modules to test: %s" % tested_addons_list)
```

Comment:
Don't change this to the list, but put the comma separated variable (the previous one).

Reply:
Well then, I will update the PR by putting all outputs to comma separated string format. But I prefer the list output because :laughing:

Reply:
OK, then keep it. I don't have a strong opinion for forcing you to change the other.

Reply:
😂 thaks